modules/biobambam/bammerge/main.nf

```diff
@@ -0,0 +1,37 @@
+process BIOBAMBAM_BAMMERGE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "bioconda::biobambam=2.0.183" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/biobambam:2.0.183--h9f5acd7_1':
+        'quay.io/biocontainers/biobambam:2.0.183--h9f5acd7_1' }"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("${prefix}.bam")    ,emit: bam
+    tuple val(meta), path("*.bam.bai")        ,optional:true, emit: bam_index
```

Comment:
There's no support for the md5checksum files

Reply:
(tentatively) fixed!